internal/transform/account_signer.go

```diff
@@ -0,0 +1,43 @@
+package transform
+
+import (
+	"fmt"
+	"github.com/guregu/null"
+	"github.com/stellar/go/ingest"
+	"github.com/stellar/stellar-etl/internal/utils"
+	"sort"
+)
+
+//TransformSigners converts account signers from the history archive ingestion system into a form suitable for BigQuery
+func TransformSigners(ledgerChange ingest.Change) ([]AccountSignerOutput, error) {
+	var signers []AccountSignerOutput
+
+	ledgerEntry, changeType, outputDeleted, err := utils.ExtractEntryFromChange(ledgerChange)
+	if err != nil {
+		return signers, err
+	}
+	outputLastModifiedLedger := uint32(ledgerEntry.LastModifiedLedgerSeq)
+	accountEntry, accountFound := ledgerEntry.Data.GetAccount()
+	if !accountFound {
+		return signers, fmt.Errorf("could not extract signer data from ledger entry of type: %+v", ledgerEntry.Data.Type)
+	}
+	sponsors := accountEntry.SponsorPerSigner()
```

Comment:
Can there be multiple sponsors on an account? And if so, how are we parsing each sponsor individually?

Reply:
I think there can be up to one sponsor per signer, and multiple signers per account.  The sponser per signer is extracted in the loop on L27.